contributing/getting-started-quick.md

```diff
@@ -62,6 +62,7 @@ This Quick Start guide is the TL;DR version of the longer [end-to-end guide](get
 # Create commits to contain your changes
 
 * Edit files in your favorite editor
+* If you edited `package.json` run `npm prune && npm install` to generate an updated `package-lock.json` file
```

Comment:
nit: Add a comma after "If you edited `package.json`"

Reply:
Done